src/smb_share.c

```diff
@@ -317,6 +341,13 @@ int             smb_share_get_list(smb_session *s, smb_share_list *list, size_t
 
     // Is the server throwing pile of shit back at me ?
     res = smb_session_recv_msg(s, &resp);
+    if (resp.payload_size < 69)
```

Comment:
It seems payload[70] is accessed below, so the minimal size should probably be 71 instead

Reply:
Yes! Thanks. I will fix this bug